integration_tests/client/metrics/test_text_generation.py

```diff
@@ -0,0 +1,708 @@
+""" These integration tests should be run with a back end at http://localhost:8000
+that is no auth
+"""
+
+from typing import Dict, List, Optional, Union
+from unittest.mock import patch
+
+import pytest
+
+from valor import (
+    Annotation,
+    Client,
+    Dataset,
+    Datum,
+    GroundTruth,
+    Model,
+    Prediction,
+)
+from valor.enums import EvaluationStatus, MetricType
+
+PREDICTIONS = [
+    """Based on the provided context, John Adams and Alexander Hamilton did not get along. John Adams, during his presidency, had grown independent of his cabinet, often making decisions despite opposition from it. Hamilton, who was accustomed to being regularly consulted by Washington, sent Adams a detailed letter with policy suggestions after his inauguration, which Adams dismissively ignored.\n""",
+    """Yes, Lincoln won the election of 1860. He received the highest number of votes and a majority in the Electoral College, making him the 16th President of the United States. However, it's important to note that he won entirely due to his support in the North and West, as he did not receive any votes in 10 of the 15 Southern slave states.""",
+    """If a turtle egg was kept warm, it would likely hatch into a baby turtle. The sex of the baby turtle would be determined by the incubation temperature, assuming the species is one of those that determine sex thermally. This is because many turtle species have the ability to move around inside their eggs to select the best temperature for development, which can influence their sexual destiny.""",
+]
+
+
+def mock_ClientConnection_evaluate(
```

Comment:
why are we mocking this instead of using the real backend (which is the point of the integration tests)?